pom.xml

```diff
@@ -35,7 +35,7 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <ddf.version>2.13.8</ddf.version>
+        <ddf.version>2.13.9-SNAPSHOT</ddf.version>
```

Comment:
❗️I think this PR should be manually heroed to make sure that this version bump didn't break anything.

Reply:
I agree. To be more specific, I think doing a quick manual smoke check. i.e. install, load the AdminUI, change a config, ingest and query in Intrigue.

Reply:
Good news! I just bumped it again